draft-ietf-tls-tls13.md

```diff
@@ -1532,55 +1532,68 @@ derivation scheme for all cipher modes.
 The basic TLS Handshake for DH is shown in {{tls-full}}:
 
 ~~~
-     Client                                               Server
-
-     ClientHello
-       + KeyShare              -------->
-                                                     ServerHello
-                                                      + KeyShare
-                                           {EncryptedExtensions}
-                                          {ServerConfiguration*}
-                                                  {Certificate*}
-                                           {CertificateRequest*}
-                                            {CertificateVerify*}
-                               <--------              {Finished}
-     {Certificate*}
-     {CertificateVerify*}
-     {Finished}                -------->
-     [Application Data]        <------->      [Application Data]
-
-            +  Indicates extensions sent in the
-               previously noted message.
-
-            *  Indicates optional or situation-dependent
-               messages that are not always sent.
-
-            {} Indicates messages protected using keys
-               derived from the ephemeral secret.
-
-            [] Indicates messages protected using keys
-               derived from the master secret.
+       Client                                               Server
+  
+Key  ^ ClientHello                                                  
+Exch v  + KeyShare               -------->                           
```

Comment:
Because it shows the server's application data after the server has received the Finished.